server/src/Metrics.hs

```diff
@@ -25,21 +25,25 @@ countHttpRequest method status httpRequestCounter = withLabel httpRequestCounter
 
 
 data IcepeakMetrics = IcepeakMetrics
-  { icepeakMetricsRequestCounter  :: HttpRequestCounter
-  , icepeakMetricsDataSize        :: Gauge
-  , icepeakMetricsJournalSize     :: Gauge
-  , icepeakMetricsDataWritten     :: Counter
-  , icepeakMetricsJournalWritten  :: Counter
-  , icepeakMetricsSubscriberCount :: Gauge
+  { icepeakMetricsRequestCounter    :: HttpRequestCounter
+  , icepeakMetricsDataSize          :: Gauge -- TODO: can be removed after dashboard has been updated to use icepeak_data_size_bytes
```

Comment:
You can put the TODOs above each line to avoid the long lines.

Reply:
I moved the comments to their own lines.